src/index.html

```diff
@@ -11,6 +11,35 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <main class="main">
+      <label for="input-big" class="main__label main__label--big">
+        <img
+          src=".//images/Search.svg"
+          alt="logo-search"
+          class="img img--big"
+        >
+        <input
+          class="main__input main__input--big"
+          type="text"
+          placeholder="Try &#34;Los Angeles&#34;"
+          id="input-big"
+          data-qa="big"
+        >
+      </label>
+      <label for="input-small" class="main__label main__label--small">
+        <img
+          src=".//images/Search.svg"
```

Comment:
The image source path is incorrect. It should not have two slashes after the dot.